api/configuration/v1/kongconsumer_types.go

```diff
@@ -89,7 +89,8 @@ func (c *KongConsumer) SetConditions(conditions []metav1.Condition) {
 
 type KongConsumerSpec struct {
 	// ControlPlaneRef is a reference to a ControlPlane this Route is associated with.
```

Comment:
Can we make explicit that this field is intended to reference Konnect entities? The same applies to all the others

Reply:
I'd say that we shouldn't do this because we might change the supported types in the ref (e.g. explicit KIC reference).